tests/integration/cattletest/core/test_apikey.py

```diff
@@ -1,13 +1,13 @@
 from common_fixtures import *  # NOQA
 
 
-def test_api_key_create(client):
-    assert_required_fields(client.create_api_key)
+def test_api_key_create(admin_client):
```

Comment:
I'm concerned that we are completely losing the ability to test that that a user can create an ApiKey. Maybe add a test for the user version of this as well?

Overkill @ibuildthecloud ?